docs/annotations/index.md

```diff
@@ -0,0 +1,234 @@
+# EnableSdaPlatform
```

Comment:
The folder and the name of your headline suggest that we still provide these annotations. Instead of "annotation" I would use the term "features", for example.

I think it would make sense to move the documentation to the respective folders that we already have, e.g. move the documentation of `EnableSdaWebSecurity` to "docs/security".

Reply:
Will update it :)